internal/multicluster/internal/types/exported_services_ent_test.go

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+//go:build consulent
```

Comment:
It looks like the majority of these tests are repeated in the ent and ce files. We should put the shared tests into internal/multicluster/internal/types/exported_services_test.go and not have any build tags on that file.